lib/create-contentful-api.ts

```diff
@@ -51,13 +50,14 @@ import {
   isClientWithLinkResolutionAndWithUnresolvableLinks,
   isClientWithoutLinkResolution,
 } from './utils/client-helpers'
+import { validateLocaleParam, validateResolveLinksParam } from './utils/validate-params'
 
 const ASSET_KEY_MAX_LIFETIME = 48 * 60 * 60
 
 export interface ClientWithLinkResolutionAndWithUnresolvableLinks extends BaseClient {
   getEntry<Fields extends FieldsType>(
     id: string,
-    query?: EntriesQueries<Fields>
```

Comment:
Why don't we need <Fields> here anymore?

Reply:
The query had the wrong type (the one for `getEntries`, but this is `getEntry`)